SeoPresentation.php

```diff
@@ -136,14 +136,9 @@ public function addExtractor(ExtractorInterface $extractor, $priority = 0)
     }
 
     /**
-     * Gets the SeoMetadata based on the content that contains the content.
-     *
-     * @param object $content
-     *
-     * @throws Exception\InvalidArgumentException
-     * @return SeoMetadata
+     * {@inheritDoc}
      */
-    private function getSeoMetadata($content)
+    public function getSeoMetadata($content)
```

Comment:
I am not really sure if that is a BC-Break? Otherwise i would implement a little wrapper method, which calls that. But calling that method would use the complete seo metadata creation including the caching for the extractors.